### ISSUE TITLE
librespot: add option to enable/disable verbose output (and maybe disable it by default)

### ISSUE THREAD

**Issue by tgurr** [badaix/snapcast#874]
The log/journal can grow quite big if not searching for a problem as it carries quite much redundant information since currently
https://github.com/badaix/snapcast/blob/be9c15cba60b64f4285ac718fa2deb1ebaf00e22/server/streamreader/librespot_stream.cpp#L72
hardcodes `--verbose` which enables e.g. `DEBUG` and `TRACE` output. It would be nice to not have this hardcoded and have an option available to configure verbose output on demand (and maybe have it disabled by default).

**Comment by badaix** [comment]
Valid point, but there is some meta-data extracted from librespot's log, I don't exactly know from what messages are used. Also there is the (outdated, default disabled) watchdog mechanism, expecting regularly incoming log messages. 
Snapcast now parses the librespot messages and logs them using the original log severity, instead of just using "Info"-level (b49016b), so all log messages are internally still available to the watch dog and the meta data reader, and the user can use the normal log message filtering (on command line or in `snapserver.conf`) to enable or disable debug logging (default off):

Before:
```
2021-05-09 09-25-48.651 [Info] (LibrespotStream) (Spotify) [2021-05-09T07:25:48Z DEBUG librespot_playback::player] command=Load(SpotifyId...
```

After (only visible with `--logging.filter=*.debug`:
```
2021-05-09 09-25-48.651 [Debug] (librespot_playback::player) command=Load(SpotifyId...
```

**Comment by badaix** [comment]
Released in [Snapcast v0.25.0](https://github.com/badaix/snapcast/releases/tag/v0.25.0)
